Rakefile

```diff
@@ -4,3 +4,10 @@ Yast::Tasks.configuration do |conf|
   #lets ignore license check for now
   conf.skip_license_check << /.*/
 end
+
+task :doc do
+  rm_rf "autodoc"
+  mkdir "autodoc"
```

Comment:
In all the other repositories, the name is "autodoc**s**". Please rename it here.